docs/tutorials/rerank/rerank_pipeline_with_bge-rerank-m3-v2_model_deployed_on_Sagemaker.md

```diff
@@ -217,18 +323,14 @@ By default, the SageMaker model output has the following format:
     "index": 1,
     "score": 0.000593021
   },
-  {
-    "index": 2,
-    "score": 0.92879725
-  },
   {
     "index": 3,
     "score": 0.00012148176
   }
 ]
 ```
 
-The connector `post_process_function` transforms the model's output into a format that the [Reranker processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/rerank-processor/) can interpret. This adapted format is as follows:
+The connector `post_process_function` transforms the model's output into a format that the [Reranker processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/rerank-processor/) can interpretm, and order result by index. This adapted format is as follows:

```

Comment:
Nit, `interpretm` should be `interpret`?

Reply:
Thank you for review. I fixed it on fixed up commit.